scripts/cleanup/cleanup_all.sh

```
@@ -243,15 +245,26 @@ for operator in "${operators[@]}"; do
         log_info "Deleting subscription for ${operator}"
         oc delete sub "$operator" --force --grace-period=0 -n openshift-operators  || true
         # Also check for subscriptions in operator-specific namespaces
+        oc delete sub "$operator" --force --grace-period=0 -n cert-manager-operator  || true
         oc delete sub "$operator" --force --grace-period=0 -n openshift-kueue-operator  || true
         oc delete sub "$operator" --force --grace-period=0 -n openshift-keda  || true
 
         log_info "Deleting CSV for ${operator}"
         oc delete csv -n openshift-operators $(oc get csv -n openshift-operators | grep "$operator" | awk '{print $1}')  || true
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)